packages/pilat/pilat.1.2/opam

```diff
@@ -0,0 +1,12 @@
+opam-version: "1.2"
+maintainer: "Steven De Oliveira <de.oliveira.steven@gmail.com>"
+authors: "Steven De Oliveira <de.oliveira.steven@gmail.com>"
+homepage: "https://github.com/Stevendeo/Pilat/"
+license: "LGPLv2.1"
+dev-repo: "https://github.com/Stevendeo/Pilat.git"
+bug-reports: "https://github.com/Stevendeo/Pilat/issues"
+build: ["sh" "make.sh"]
+install: ["sh" "install.sh"]
+remove: ["ocamlfind" "remove" "pilat"]
+depends: ["ocamlfind" "lacaml" "zarith" "frama-c"]
```

Comment:
Could you flag `ocamlfind` as a `{build}` dependency if Pilat doesn't use the `Findlib` library ?